hal/src/main/native/athena/HAL.cpp

```diff
@@ -357,6 +358,12 @@ HAL_Bool HAL_Initialize(int32_t timeout, int32_t mode) {
 
   HAL_InitializeDriverStation();
 
+  // Set WPI_Now to use FPGA timestamp
+  wpi::SetNowImpl([]() -> uint64_t {
+    int32_t status = 0;
+    return HAL_GetFPGATime(&status);
+  });
```

Comment:
https://github.com/wpilibsuite/allwpilib/blob/master/hal/src/main/native/shared/cpp/fpga_clock.cpp#L22

Just like we do in fpga_clock, we should print an error on a status failure.